rollup.config.js

```diff
@@ -9,6 +10,10 @@ const stylesheet = require('./rollup.plugins').stylesheet;
 const pkg = require('./package.json');
 
 const input = 'src/index.js';
+
+// regexp for file exlusinon for babel. we are excluding from the exclusion the modules that need to be transpiled
+const exclude = /node_modules\/(?!(@kurkle\/color)\/).*/;
```

Comment:
Do we even need exclusions now? `color` is the only dependency the library has

Reply:
Turns out no, so removed.